_drafts/mastering-with-git.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Mastering git with team"
+date:   2016-12-6 12:55:28 +0100
+categories: devops project management versioning workflow
+---
+## Introduction
+Work on project with people must be controlled by rules, systems, and best
+pratices work ensure the long life of the project.
+
+Working on project can be addressed to different publics like, a complet project team (private or free projects), 
+simple user who want to contribute to an upstream project, or also core team of a project who want work with external contributors.
+
```

Comment:
Working on a project may address serveral cases. It could be a single developper who needs to version his code, a working team which needs to work together easily, or an open project which let the capability to contribute to other developpers than the core team.